tools/stern/src/Stern/API.hs

```diff
@@ -337,22 +337,21 @@ routes = do
       description "Team ID"
     Doc.parameter Doc.Path "feature" Public.typeTeamFeatureName $
       description "Feature name"
-    Doc.returns Public.typeTeamFeatureStatus
+    Doc.returns (Doc.ref Public.modelTeamFeatureStatus)
```

Comment:
I'd also go for the value here, as you intended, but this way we keep strings consistently out of http bodies.